tests/integration/src/utils/cdk.rs

```diff
@@ -0,0 +1,80 @@
+use std::{path::Path, process::Command, time::Duration};
+
+use anyhow::Result;
+use async_std::task;
+use log::info;
+use once_cell::sync::Lazy;
+
+use crate::utils::connector_dir;
+
+static CDK_BIN: Lazy<String> = Lazy::new(|| std::env::var("CDK_BIN").unwrap_or("cdk".to_string()));
+
+pub async fn cdk_deploy_start(config_path: &Path, env: Option<(&str, &str)>) -> Result<()> {
+    let connector_dir = connector_dir()?;
+    info!(
```

Comment:
should use structure format:
```
info!(config_path,"displaying connector")...
```

Reply:
we're using log here, not tracing lib. I think log does not support structure format. 
I removed btw, wdyt?
I can also replace log in a new PR.